client/src/Components/Mentors.js

```diff
@@ -5,11 +5,14 @@ import "../App.css";
 import "../grid.css";
 import ReactMarkdown from "react-markdown";
 import RunQuiz from "./RunQuiz";
+import Modal from "../Modal/Modal";
 export default function Mentors(props) {
 	const [newQuizQuestions, setNewQuizQuestions] = useState([]);
 	const [numberOfQuestions,setNumberOfQuestions]=useState(5);
 	const [filteredQuestionsByTag, setFilteredQuestionsByTag]=useState(props.questions);
 	const [tagsCollection, setTagsCollection] = useState([]);
+	const [response, setResponse]=useState(null);
```

Comment:
Do we need this extra state? Can we just `setModalText` if `response.statusText !== "OK"` below?

Reply:
Absolutely agree. This was actually the first iteration and evolved later in setModalText and extra state needs o bee removed thank you for notice